reporter/html.js

```diff
@@ -896,4 +897,27 @@ if ( notPhantom && document.readyState === "complete" ) {
 	addEvent( window, "load", QUnit.load );
 }
 
+// Wrap window.onerror. We will call the original window.onerror to see if
+// the existing handler fully handles the error; if not, we will call the
+// QUnit.onError function.
+var originalWindowOnError = window.onerror;
+
+// Cover uncaught exceptions
+// Returning true will suppress the default browser handler,
+// returning false will let it run.
+window.onerror = function( ...args ) {
+	var ret = false;
+	if ( originalWindowOnError ) {
+		ret = originalWindowOnError( ...args );
```

Comment:
This should be `ret = originalWindowOnError.apply( this, args )` for strict adherence to [the spec](https://html.spec.whatwg.org/multipage/webappapis.html#onerroreventhandler):
> …and with the [_callback this value_](https://heycam.github.io/webidl/#dfn-callback-this-value) set to _E_'s [`currentTarget`](https://dom.spec.whatwg.org/#dom-event-currenttarget).

It will probably never matter, but environments can be faked and being explicit doesn't hurt.